osu.Game/Graphics/OsuColour.cs

```diff
@@ -156,13 +156,13 @@ public Color4 ForModType(ModType modType)
                     return Lime1;
 
                 case ModType.Conversion:
-                    return Purple1;
+                    return Purple0;
 
                 case ModType.Fun:
-                    return Pink1;
+                    return Pink0;
```

Comment:
These changes have significant impact on how the mod select screen looks, to the point of deviating from [design](https://www.figma.com/file/NkdREXr6wFkA99oLw8PnBv/Client%2FMod-Selection?node-id=0%3A1&t=RqaWGABSXQwEGxYh-0):

| before | after |
| :-: | :-: |
| ![1673652505](https://user-images.githubusercontent.com/20418176/212436987-3f51252b-252e-4310-9269-526ec429219d.png) | ![1673652379](https://user-images.githubusercontent.com/20418176/212436995-c93526bf-4d70-4fd8-8305-47bf92a88b1c.png) |

The version after the changes looks worse to me to be frank. Too pastelly.

Reply:
yep, old colours look better to my eye.